### ISSUE TITLE
Check executable path when executing `rvc` with `sudo`

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#185]
@jjr840430 can you please implement a check in `rvc` that checks whether the `rvc` binary is executed from: macOS =`/opt/rvc/bin` and Linux = `/usr/local/bin`.

For Linux you can `fopen()` `/proc/self` to see what argv[0] is and on macOS you can use `_NSGetExecutablePath`.

This enhancement is to make sure `rvc` is not executed with `sudo` from other directories than intended and it will force macOS users to insert `/opt/rvc/bin` into their `PATH` or else things won't work.

I think the check can be done at the same place as `getuid() !=0` but you might come up with a better solution/place:
```$ grep -rin getuid *
src/rvc_shared.c:542:	if (getuid() != 0) {
src/rvc_shared.c:574:	if (getuid() != 0) {
src/rvc_shared.c:652:	if (getuid() != 0) {
src/rvc_shared.c:729:	if (getuid() != 0) {
src/rvc_shared.c:763:	if (getuid() != 0) {
```

Thanks

**Comment by jjr840430** [comment]
@erikbor Fixed by #190 #189

**Comment by erikbor** [comment]
@jjr840430 this is excellent!!! thank you :D
